addons/sale_product_configurator/static/src/js/tours/sale_product_configurator.js

```diff
@@ -1,18 +0,0 @@
-odoo.define("sale_product_configurator.tour", function (require) {
-"use strict";
-/**
```

Comment:
why is this no longer needed?

Reply:
The sale tour was fixed to take either the product_id or product_template_id for its tour.
So this "hack" isn't needed anymore